warn/warn_cosmetic_test.go

```diff
@@ -185,6 +185,33 @@ package()
 foo(baz)`,
 		[]string{":11: Package declaration should be at the top of the file, after the load() statements, but before any call to a rule or a macro. package_group() and licenses() may be called before package()."},
 		scopeDefault|scopeBzl|scopeBuild)
+
+	checkFindingsAndFix(t,
```

Comment:
Add a test case for when variable is NOT used in package statement?
(mainly to document the expected behavior)

Reply:
Done.